<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" rowspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(38,188,213);
color: rgb(0, 0, 0);
border-radius:10px;
font:18pt;
font:bold;
font-family:宋体;</string>
           </property>
           <property name="text">
            <string> 用户名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="userLineEdit">
           <property name="styleSheet">
            <string notr="true">
QLineEdit{
background-color: rgb(255,255,255);
color: rgb(0, 0, 0);
border-radius:10px;
font:15pt;
font-family:Microsoft Yahei;
border:1px solid gray;
}
QLineEdit:hover{
border-width:2px;
border-color:rgb(170, 170, 255);
}
QLineEdit:focus {
border-width:2px;
border-color:rgb(170, 170, 255);
}</string>
           </property>
           <property name="placeholderText">
            <string>请输入用户名</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(38,188,213);
color: rgb(0, 0, 0);
border-radius:10px;
font:18pt;
font:bold;
font-family:宋体;</string>
           </property>
           <property name="text">
            <string> 密  码：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordLineEdit">
           <property name="contextMenuPolicy">
            <enum>Qt::ActionsContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">
QLineEdit{
background-color: rgb(255,255,255);
color: rgb(0, 0, 0);
border-radius:10px;
font:15pt;
font-family:Microsoft Yahei;
border:1px solid gray;
}
QLineEdit:hover{
border-width:2px;
border-color:rgb(170, 170, 255);
}
QLineEdit:focus {
border-width:2px;
border-color:rgb(170, 170, 255);
}</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>请输入密码</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>32</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="cancelPushButton">
           <property name="styleSheet">
            <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:10px;
	font: 19pt;
}

QPushButton:hover
{
    background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
    background-color:rgb(14 , 100 , 228);
    padding-left:2px;
    padding-top:2px;
}
QPushButton:unpressed
{
    background-color:rgb(0 , 100 , 0);
    padding-left:2px;
    padding-top:2px;
}</string>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="loginPushButton">
           <property name="styleSheet">
            <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:10px;
	font: 19pt;
}

QPushButton:hover
{
    background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
    background-color:rgb(14 , 100 , 228);
    padding-left:2px;
    padding-top:2px;
}
QPushButton:unpressed
{
    background-color:rgb(0 , 100 , 0);
    padding-left:2px;
    padding-top:2px;
}</string>
           </property>
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font:22pt;
font:bold;
border-radius:10px;
</string>
       </property>
       <property name="text">
        <string> 智能集中控制器远程管理系统 </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>90</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>15</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>15</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>59</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
